tournaments/generator-round-robin.js

```diff
@@ -1,31 +1,48 @@
 'use strict';
 
+/** @typedef {{state: string, score: number[], result: string}} Match */
```

Comment:
Maybe `score?` and `result?` – it seems better than changing the return signature.